app/api/utility.py

```diff
@@ -51,22 +51,11 @@ def create_query(age: tuple = (None, None), sex: str = None) -> str:
     """
     subject_level_filters = ""
 
-    if (
-        isinstance(age, tuple)
-        and not age == (None, None)
-        and not age == ("", "")
-    ):
-        # TODO: revisit this and replace this solution with one that just doesn't add the filter condition.
-        age = tuple(
-            (
-                default_val if age_val is None else age_val
-                for age_val, default_val in zip(age, [0, 100])
-            )
-        )
-        subject_level_filters += (
-            "\n"
-            + f"FILTER (?{AGE.var} >= {age[0]} && ?{AGE.var} <= {age[1]})."
-        )
+    if isinstance(age, tuple):
```

Comment:
what should happen if `age` is not a tuple? Probably out of scope for this PR. But with the current setup, we could have a silent failure if we somehow pass a non-tuple (e.g. another iterable like a list) by accident. Then we think we filter age, but we don't. One option would be to remove the `if` here and accept that we may get an `IndexError` if we somehow just pass an `int`. Another option would be to manually raise a `TypeError` if it is not `tuple`. 

Afaik, only our own code will ever call this function. So we could decide to not handle "user error" gracefully and instead just crash. Then we'll probably catch a mistake like "we thought we could pass `age=12` to `create_query` and get away with it" because our tests will be failing (hopefully). 

We should probably do this in a separate issue though because the code here is working and we don't want to drag this out!